### ISSUE TITLE
Not all EPUB files are displayed

### ISSUE THREAD

**Issue by Avelle1** [rhaleblian/dslibris#38]
I added the app into my New 2DS XL. everything is on the root directory and the app works fine. the only problem is that only a few of the epub files are in the library and I can read them. the rest aren't showing, I am not sure what else I need to do. does the app only store a certain amount of epub files? or is there something else that I am missing. i have attach the log so maybe there's something there that I need to do?
[dslibris.log](https://github.com/rhaleblian/dslibris/files/13196894/dslibris.log)

**Comment by rhaleblian** [comment]
Uhoh.
There may be a limit on the number of epubs, I have never put more than a dozen on my storage.  How many do you have?

**Comment by Avelle1** [comment]
A dozen huh, for me it only put in 9 on the library, and I put it about 50 of it in storage. What’s odd tho is that when i booted it up, it scans all of it, not only just those 9.

**Comment by rhaleblian** [comment]
> it scans all of it, not only just those 9.
Yes, in the sense that the log reposts that many more than 9 books were found.

Is at least a second page of books being displayed in the initial book browser?

**Comment by Avelle1** [comment]
Yes it did show a second page of books, the first showed 6 of the 9 books that were initially shown, and the second page showed the rest of the 9

**Comment by rhaleblian** [comment]
This was likely a particular book.  We are lucky dslibris didn't crash actually.
